### PR TITLE
Avoid use of ideep in doctest

### LINKED ISSUE #5134: doctest requires ideep4py
doctest of `docs/source/tips.rst` fails without ideep4py installed.
We should either require it explicitly or fix the test.

### PR DESCRIPTION
**Pull request by kmaehashi**
This fixes #5134.
Please merge https://github.com/chainer/chainer-test/pull/441 after this PR and backport got merged.

### CODE REVIEW

**Comment by kmaehashi** [comment]
Jenkins, test this please.

File: docs/source/tips.rst [hunk 1]
```diff
@@ -141,10 +141,16 @@ You can also use :func:`chainer.using_config` to change the configuration.
         y = chainer.functions.relu(x)
     print(type(y.data))
 
-.. testoutput::
+.. code-block:: console
```
Comment (niboshi) [inline docs/source/tips.rst]: How about `none`, which is more consistent with what would be generated by `testoutput`?

Comment (kmaehashi) [reply docs/source/tips.rst]: OK, fixed.

**Comment by niboshi** [comment]
Jenkins, test this please.

**Comment by niboshi** [comment]
LGTM

**Pull request merged**